TombEngine/Game/Lara/lara_flare.cpp

```diff
@@ -351,7 +353,7 @@ void CreateFlare(ItemInfo& laraItem, GAME_OBJECT_ID objectID, bool isThrown)
 
 	flareItem.Pose.Orientation.x = 0;
 	flareItem.Pose.Orientation.z = 0;
-	flareItem.Model.Color = Vector4(0.5f, 0.5f, 0.5f, 1.0f);
+	flareItem.Model.Color = Vector4::One;
```

Comment:
What issue were the old values causing?

Reply:
When the torch was launched, the new object looked darker, (in comparison with the torch object that Lara was holding, or the one pickable from the ground).

I think TR1 and TR2 had the 0.5f grey as default, but TR3 forward objects get the 1.0f white as default and from there they get coloured.  I think is better correct the 0.5's to 1.0's instead to change all the 1.0's back int 0.5's